connectors/src/connectors/slack/bot.ts

```diff
@@ -770,11 +772,29 @@ async function makeContentFragment(
   // Prepend $url to the content to make it available to the model.
   const section = `$url: ${url}\n${sectionFullText(content)}`;
 
+  const contentType = "dust-application/slack";
+  const fileName = `slack_thread-${channel.channel.name}.txt`;
+
+  const fileRes = await dustAPI.uploadFile({
+    contentType,
+    fileName,
+    fileSize: section.length,
+    useCase: "conversation",
+    fileObject: new File(
+      [new Blob([section], { type: contentType })],
```

Comment:
IIRC, string don't need to converted to Blob firs ([doc](https://developer.mozilla.org/en-US/docs/Web/API/File/File))
Simply passing section here should work.

Reply:
fixed :+1: